### DISCUSSION TITLE
Offer to translate documentation

### DISCUSSION THREAD

**Discussion by eliasdorneles** [Textualize/textual#3109] [General]
Hello,

I'm interested in helping to translate the documentation, so that more people can use Textual.
I'm notably interest on translating into French and Brazilian Portuguese.

I might know some other people that will be interested on this task.

Is this a contribution that the project is open to?

It would require some setup for Mkdocs: https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/

**Comment by LrsNate** [comment]
Seconded! I'd love to help translate to French :)
